### ISSUE TITLE
Allow reset to default defence configurations

### ISSUE THREAD

**Issue by heatherlogan-scottlogic** [gsproston-scottlogic/prompt-injection#340]
When a user changes the configurations in sandbox, they cannot get back to the default setting.
This would be useful especially for long system roles/prompts.

Options
1. add a reset button to each configuration e.g. 

![Image](https://github.com/ScottLogic/prompt-injection/assets/118981273/05cb299e-9715-457c-b99d-b607c9ad69be)

GIVEN i edit a defence configuration
WHEN i click the reset button
THEN that defence config should go back to default
